apps/innbyggerpanelet/src/app/containers/candidateProfile/CandidatePerformedInsight.tsx

```diff
@@ -0,0 +1,17 @@
+import { Insight } from '@innbyggerpanelet/api-interfaces'
+import { Heading } from '@navikt/ds-react'
+import React, { ReactElement } from 'react'
+import PropertieValueField from '../../components/misc/propertieValueField/PropertieValueField'
+
+import style from './CandidatePerformedInsight.module.scss'
+
+export default function CandidatePerfomedInsight(insightWorks: Insight[]): ReactElement {
+    return (
+        <div>
+            <div className={style.candidatePerfomedInsight}>
+                <Heading size='large'>Tidligere innsiktsarbeid</Heading>
+                {Object.keys(insightWorks).map((item, i) => <PropertieValueField key={i} propertie={insightWorks[i].name} value={insightWorks[i].insightTechnique}/>)}
```

Comment:
Should be able to access .map() directly from insightWorks, without calling Object.keys().

Reply:
Unless the intention is to abstract away any implicit types?

Reply:
For some odd reason typescript would not recognize my array as an array, even with a type declaration. It instead recognized the array as an object, forsing me to convert the "object" to an array

Reply:
Messy future bug, will look more into it